dashboard/src/actions/overviewActions.js

```diff
@@ -49,7 +49,10 @@ export const getDatasets = () => async (dispatch, getState) => {
       dispatch({ type: TYPES.OPENID_ERROR });
       clearCachedSession(dispatch);
     } else {
-      dispatch(showToast(DANGER, error?.response?.data?.message));
+      const msg = error.response?.data?.message;
+      dispatch(
+        showToast(DANGER, msg ? msg : `Error response: ${error.response}`)
```

Comment:
> How will Javascript format a Response object?

It's a valid question which I decided to punt on the principal of the perfect being the enemy of the done.  This is something that we can iterate on.

> Is that really what we want to appear in the toast?

Probably, not; but my instinct was that it was better than a generic (local fixed-text) message.

> I'm not sure how "clever" we want to be here

Only as clever as we need to be and no cleverer.  If you're reasonably sure that there will be a `response.reason`, I'm OK with using that instead of the whole `response` (I was hoping that Javascript would do something reasonable, and that having the whole response might be be illuminating).  If we use the chaining dereference, I'm OK with seeing an `undefined` in the rare case that we get an error with no other information (i.e., that's not much different from getting a literal "something went wrong" message....)